doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -256,7 +256,10 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` service:
```

Comment:
`bt_fast_pair_readme` would be displayed as `Google Fast Pair Service (GFPS)`. There is no need to duplicate word `service` here

Reply:
I see. It is copied from NCS 2.3.0 release notes but I will change it here.